kahuna/public/js/search/results.html

```diff
@@ -13,14 +13,43 @@
 
     <div ng:if="ctrl.totalResults > 0">
 
-        <div class="image-results-count">
-            {{ctrl.totalResults | toLocaleString}} matches
-            <span ng:if="ctrl.inSelectionMode()">({{ctrl.selectedImages.size}} selected)</span>
-            <button ng:show="ctrl.newImagesCount > 0"
-                    ng:click="ctrl.revealNewImages()"
-                    class="image-results-count__new">
-                {{ctrl.newImagesCount | toLocaleString}} new
-            </button>
+        <div class="results-toolbar">
+            <div class="image-results-count">
+                {{ctrl.totalResults | toLocaleString}} matches
+                <span ng:if="ctrl.inSelectionMode()">({{ctrl.selectedImages.size}} selected)</span>
+                <button ng:show="ctrl.newImagesCount > 0"
+                        ng:click="ctrl.revealNewImages()"
+                        class="image-results-count__new">
+                    {{ctrl.newImagesCount | toLocaleString}} new
+                </button>
+            </div>
+            <div class="sort-direction">
+                <div class="radio-list">
+                    <div class="radio-list__item">
+                        <input type="radio"
+                               id="sort-direction__desc"
+                               class="radio-list__circle"
+                               name="sort-direction"
+                               ng:value="undefined"
+                               ng:model="ctrl.filter.orderBy">
+                        <label for="sort-direction__desc" class="radio-list__label">
+                            Newest first
+                        </label>
+                    </div>
+
+                    <div class="radio-list__item">
+                        <input type="radio"
+                               id="sort-direction__asc"
+                               class="radio-list__circle"
+                               name="sort-direction"
+                               ng:value="'oldest'"
```

Comment:
Query param becomes `orderBy=oldest`, with the hope that we'll soon support something like:

`orderOn=<uploadTime|lastModified|dateTaken>`